CompilerSource/compiler/components/write_room_data.cpp

```diff
@@ -198,7 +198,7 @@ wto.open((codegen_directory + "Preprocessor_Environment_Editable/IDE_EDIT_roomcr
     {
       wto << "variant room_"<< es->rooms[i].id <<"_instancecreate_" << it->first << "()\n{\n  ";
       if (mode == emode_debug) {
-        wto << "enigma::debug_scope $current_scope(\"'instance creation' for instance '" << it->first << "'\");\n";
+        wto << "enigma::debug_scope $current_scope(\"'instance creation' for instance '" << it->first << "'\");\n  ";
```

Comment:
Are you sure? What about line 199? That is where I copied the style from, because it seems to be what you were already doing. Unless, you are not the one who wrote line 199.

I went and looked at the source back about 8 years and it seems this has been the way it always was in various places:
https://github.com/enigma-dev/enigma-dev/blame/19c85d9f80eef885338dc0c3395ff1117a702c93/CompilerSource/compiler/components/write_room_data.cpp#L167

I don't think you want me to correct this now because I'd probably have to refactor this whole file and I believe we don't want to mixup those kinds of changes. Do it in a separate pr after this is merged?

Reply:
I've split these changes out into #1211, see above comment.